pyproject.toml

```diff
@@ -33,6 +33,15 @@ packages = [
 
 [project.optional-dependencies]
 dev = ["pre-commit", "pip-tools", "ruff", "pyright"]
+test = [
+    "matplotlib-stubs[dev]",
+    "mypy",
+    "numpy",
+    "pandas",
```

Comment:
Since they are used in the stubs, they should not be optional dependencies, right ?